src/app/features/ahbs/views/ahb-page/ahb-page.component.ts

```diff
@@ -58,6 +60,35 @@ export class AhbPageComponent {
     });
   }
 
+  // mapping provided by mig_ahb_utility_stack
+  private getEdifactFormat(pruefi: string): string {
+    const mapping: { [key: string]: string } = {
+      '99': 'APERAK',
+      '29': 'COMDIS',
+      '21': 'IFTSTA',
+      '23': 'INSRPT',
+      '31': 'INVOIC',
+      '13': 'MSCONS',
+      '39': 'ORDCHG',
+      '17': 'ORDERS',
+      '19': 'ORDRSP',
+      '27': 'PRICAT',
+      '15': 'QUOTES',
+      '33': 'REMADV',
+      '35': 'REQOTE',
+      '37': 'PARTIN',
+      '11': 'UTILMD',
+      '25': 'UTILTS',
+      '91': 'CONTRL',
+      '92': 'APERAK',
+      '44': 'UTILMD', // UTILMD for GAS since FV2310
+      '55': 'UTILMD', // UTILMD for STROM since FV2310
```

Comment:
denke hier können wir noch ergänzen dass es sich um Strom bzw Gas handelt

```suggestion
      '44': 'UTILMD Gas', // UTILMD for GAS since FV2310
      '55': 'UTILMD Strom', // UTILMD for STROM since FV2310
```